app/client/components/helpCentre/helpCentreContactOptions.tsx

```diff
@@ -0,0 +1,126 @@
+import { css } from "@emotion/core";
+import { Button } from "@guardian/src-button";
+import { space } from "@guardian/src-foundations";
+import { neutral } from "@guardian/src-foundations/palette";
+import { headline, textSans } from "@guardian/src-foundations/typography";
+import React, { useState } from "react";
+import { minWidth } from "../../styles/breakpoints";
+import { CallCentreEmailAndNumbers } from "../callCenterEmailAndNumbers";
+import { isLiveChatFeatureEnabled } from "../liveChat/liveChatFeatureSwitch";
+import {
+  LiveChatPrivacyNotice,
+  LiveChatPrivacyNoticeLink,
+} from "../liveChat/liveChatPrivacyNotice";
+import { HelpCentreEmailAndLiveChat } from "./helpCentreEmailAndLiveChat";
+import { HelpCentrePhoneNumbers } from "./helpCentrePhoneNumbers";
+
+interface HelpCentreContactOptionsProps {
+  compactLayout?: boolean;
+}
+
+const baseSubtitleStyles = css`
+  border-top: 1px solid ${neutral["86"]};
+  margin-top: 30px;
+  ${headline.small({ fontWeight: "bold" })};
+`;
+
+const subtitleStyles = css`
+  ${baseSubtitleStyles}
+  margin-bottom: ${space[6]}px;
+  ${minWidth.tablet} {
+    margin-bottom: ${space[6]}px;
+    margin-top: 40px;
+  }
+`;
+
+const liveChatSubtitleStyles = css`
+  ${baseSubtitleStyles}
+  margin-bottom: ${space[1]}px;
+  ${minWidth.tablet} {
+    margin-bottom: 0;
+    margin-top: 40px;
+  }
+`;
+
+const emailAndLiveChatSubheadingCss = css`
+  ${textSans.medium()};
+  margin-bottom: ${space[1]}px;
+  max-width: 320px;
+  ${minWidth.tablet} {
+    max-width: none;
+  }
+`;
+
+const emailAndLiveChatSubheadingWideCss = css`
+  ${minWidth.wide} {
+    display: none;
+  }
+`;
+
+const contactButtonCss = css`
+  margin-top: ${space[4]}px;
+  margin-bottom: ${space[9]}px;
+  ${minWidth.desktop} {
+    margin-top: ${space[6]}px;
+    margin-bottom: ${space[12]}px;
+  }
+`;
+
+const HelpCentreContactOptions = (props: HelpCentreContactOptionsProps) => {
+  const [contactOptionsVisible, setContactOptionsVisible] = useState(
+    !props.compactLayout
```

Comment:
I'm not entirely sold on `compactLayout` as a name as it's also being used here to hide the contact options initially, but I wanted a name that allowed for reuse rather than something like `articleLayout`. Perhaps being specific isn't a bad thing in this case though?

Reply:
imo a specific name here is better

Reply:
I know you've already approved the PR, but do you think I should rename this to something other than `compactLayout`?

Reply:
I'd name it to `contactOptionsVisible` or similar like you have there as the state variable name?

Reply:
I've added a separate `hideContactOptions` prop for this to decouple it from switching the layout. I think it makes sense to show the contact options by default and hiding them be the exception.

Reply:
yep, good idea